lib/shared/drive-constraints.ts

```diff
@@ -58,8 +58,10 @@ export interface Image {
  * In the context of Etcher, a source drive is a drive
  * containing the image.
  */
-export function isSourceDrive(drive: DrivelistDrive, image: Image): boolean {
-	image = image || {};
```

Comment:
LOL, I _did_ wonder why it was being done that way in the previous PR; but I didn't think it was worth mentioning as I assumed there was a reason for it being done that way :laughing:

Reply:
Gotta go fast :P